src/Components/Articles.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState, useEffect } from "react"
+import { getArticles } from "../api"
+import ArticleCard from "./ArticleCard"
+
+const Articles = () => {
+
+    const [articles, setArticles] = useState([])
+
+    useEffect(() => {
+        getArticles().then((articles) => {
+            setArticles(articles)
+        })
+    }, [])
+
```

Comment:
It would be great to see a loading pattern here. Your api will spin down after 15 mins of inactivity and take some time to spin back up. While that is spinning up a user would ideally want to know things are working 💡